From b046e44a7f444b1150ce765512dd56a6505229b2 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <xnox@debian.org>
Date: Sat, 19 Apr 2014 12:10:15 +0100
Subject: Default to acting like fsck.

Inspect arguments, if we are not called as btrfs, then assume we are
called to act like fsck.

Bug-Debian: http://bugs.debian.org/712078
Signed-off-by: Dimitri John Ledkov <xnox@debian.org>
---
 btrfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/btrfs.c b/btrfs.c
index f8b87af..adbd668 100644
--- a/btrfs.c
+++ b/btrfs.c
@@ -261,7 +261,7 @@ int main(int argc, char **argv)
 	else
 		bname = argv[0];
 
-	if (!strcmp(bname, "btrfsck")) {
+	if (strcmp(bname, "btrfs") != 0) {
 		argv[0] = "check";
 	} else {
 		argc--;
